icons/rows.json

```
@@ -0,0 +1,9 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "layout"
```

```suggestion
    "layout",
    "lines",
    "queue",
    "series",
    "list",
    "vertical",
    "horizontal"
```
(vertically listed items that have a horizontal orientation that's why both)